sourcegraph/cody
# "Fix problems" as a command in the agent API

Partially implements https://github.com/sourcegraph/jetbrains/issues/117.

## Test plan
1. Check out this PR: https://github.com/sourcegraph/jetbrains/pull/1859
2. Run this command:
   ```
   CODY_DIR=/path/to/cody ./gradlew runIde -PforceCodyBuild=true
   ```
3. Open any source file in any project
4. Select a range of lines and right-click. The new action will be available in the Cody sub-menu.
